frappe/public/js/frappe/desk.js

```diff
@@ -466,12 +466,34 @@ frappe.Application = Class.extend({
 
 	show_change_log: function() {
 		var me = this;
-		var d = frappe.msgprint(
-			frappe.render_template("change_log", {"change_log": frappe.boot.change_log}),
-			__("Updated To New Version")
-		);
-		d.keep_open = true;
-		d.custom_onhide = function() {
+		let change_log = frappe.boot.change_log;
+
+		// frappe.boot.change_log = [{
+		// 	"change_log": [
+		// 		[<version>, <change_log in markdown>],
+		// 		[<version>, <change_log in markdown>],
+		// 	],
+		// 	"description": "ERP made simple",
+		// 	"title": "ERPNext",
+		// 	"version": "12.2.0"
+		// }];
+
+		// Iterate over changelog
+		change_log.forEach(log => {
+			log.change_log.forEach(version_info => {
+				version_info[1] = version_info[1].replace(/#/, '##');
```

Comment:
This seems like a hack. 
You can either directly fix it [here](https://github.com/frappe/frappe/blob/develop/frappe/change_log/v12/v12_0_0.md) or tweak the CSS for `.app-change-log`

Reply:
The fix in markdown cannot be controlled by as much, the css tweak for `.app-change-log` seems much better! I'll use that